app/js/components/answers/Index.react.js

```diff
@@ -46,12 +46,27 @@ class AllAnswers extends React.Component {
      Common.initTinyMceContent('.answer');
    }
 
+   sortAnswers(answers_obj){
```

Comment:
I'm not sure I understand. Are you referring to the `sortAnswers()` method, or the `this.setState()` declaration ?